src/components/Login.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import '../style/Login.css';
+import { useHistory } from 'react-router-dom';
+import Logo from '../common/Logo';
+import TweetBtn from '../common/TweetBtn';
+import { login } from '../actions/loginActions'
+import { connect } from 'react-redux';
+
+
+function Login(props) {
+    const [name, setName] = useState('');
+    const [password, setPassword] = useState('');
+    const history = useHistory();
+
+    function handleNameChange(e) {
+        setName(e.target.value);
+    }
+    function handlePasswordChange(e) {
+        setPassword(e.target.value);
+    }
+
+    function handleLogin() {
+        try {
+            history.push('/home');
```

Comment:
Consts like 'home' and more alike should be in const file (just for the order and semantics)